roachtest: ruby-pg failed

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.ruby-pg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/17349052?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/17349052?buildTab=artifacts#/ruby-pg) on release-24.2.4-rc @ [8140b8e294fb16a293d0aae662bcf5c1d488c852](https://github.com/cockroachdb/cockroach/commits/8140b8e294fb16a293d0aae662bcf5c1d488c852):


```
(orm_helpers.go:229).summarizeFailed: 
Tests run on Cockroach v24.2.4-dev-8140b8e294fb16a293d0aae662bcf5c1d488c852
Tests run against ruby-pg v1.4.6
898 Total Tests Run
713 tests passed
185 tests failed
0 tests skipped
4 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: running with sync_* methods PG::Connection times out after connect_timeout seconds -  (unexpected)
For a full summary look at the ruby-pg artifacts 
An updated blocklist (rubyPGBlocklist) is available in the artifacts' ruby-pg log
test artifacts and logs in: /artifacts/ruby-pg/run_1
```

Parameters:
 - <code>ROACHTEST_arch=amd64</code>
 - <code>ROACHTEST_cloud=gce</code>
 - <code>ROACHTEST_coverageBuild=false</code>
 - <code>ROACHTEST_cpu=4</code>
 - <code>ROACHTEST_encrypted=false</code>
 - <code>ROACHTEST_runtimeAssertionsBuild=true</code>
 - <code>ROACHTEST_ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-17349052/ruby-pg/1729157497025/1729158114770)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*ruby-pg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)
</sub>


Jira issue: CRDB-43293

--- a/pkg/cmd/roachtest/tests/activerecord.go
+++ b/pkg/cmd/roachtest/tests/activerecord.go
@@ -146,7 +146,7 @@ func registerActiveRecord(r registry.Registry) {
 			c,
 			node,
 			"installing bundler",
-			`cd /mnt/data1/activerecord-cockroachdb-adapter/ && sudo gem install bundler:2.1.4`,
+			`cd /mnt/data1/activerecord-cockroachdb-adapter/ && sudo gem install bundler:2.4.9`,
 		); err != nil {
 			t.Fatal(err)
 		}

--- a/pkg/cmd/roachtest/tests/pgjdbc_blocklist.go
+++ b/pkg/cmd/roachtest/tests/pgjdbc_blocklist.go
@@ -399,7 +399,6 @@ var pgjdbcBlockList = blocklist{
 	`org.postgresql.test.jdbc2.ServerErrorTest.testNotNullConstraint`:                                                                                               "27796",
 	`org.postgresql.test.jdbc2.ServerErrorTest.testPrimaryKey`:                                                                                                      "27796",
 	`org.postgresql.test.jdbc2.StatementTest.closeInProgressStatement()`:                                                                                            "unknown",
-	`org.postgresql.test.jdbc2.StatementTest.concurrentWarningReadAndClear()`:                                                                                       "unknown",
 	`org.postgresql.test.jdbc2.StatementTest.fastCloses()`:                                                                                                          "unknown",
 	`org.postgresql.test.jdbc2.StatementTest.parsingSemiColons()`:                                                                                                   "unknown",
 	`org.postgresql.test.jdbc2.StatementTest.updateCount()`:                                                                                                         "unknown",

--- a/pkg/cmd/roachtest/tests/ruby_pg.go
+++ b/pkg/cmd/roachtest/tests/ruby_pg.go
@@ -146,7 +146,7 @@ func registerRubyPG(r registry.Registry) {
 			c,
 			node,
 			"installing bundler",
-			`cd /mnt/data1/ruby-pg/ && sudo gem install bundler:2.1.4`,
+			`cd /mnt/data1/ruby-pg/ && sudo gem install bundler:2.4.9`,
 		); err != nil {
 			t.Fatal(err)
 		}

